src/applications/appeals/995/components/EvidencePrivateRecordsAuthorization.jsx

```diff
@@ -90,6 +90,7 @@ const EvidencePrivateRecordsAuthorization = ({
           label={authorizationLabel}
           checked={data.privacyAgreementAccepted}
           onVaChange={handlers.onChange}
+          aria-describedby="authorizationInfo"
```

Comment:
this should contain the ID that was added `authorize-text`

Reply:
wow. I really changed the id on myself. Good catch!

Reply:
@Mottie Should be updated now.